hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/net/TestNetUtils.java

```diff
@@ -360,6 +360,49 @@ public void testCreateSocketAddress() throws Throwable {
     }
   }
 
+  @Test
+  public void testCreateSocketAddressWithURICache() throws Throwable {
+    InetSocketAddress addr = NetUtils.createSocketAddr(
+        "127.0.0.1:12345", 1000, "myconfig", true);
+    assertEquals("127.0.0.1", addr.getAddress().getHostAddress());
+    assertEquals(12345, addr.getPort());
+
+    addr = NetUtils.createSocketAddr(
+        "127.0.0.1:12345", 1000, "myconfig", true);
+    assertEquals("127.0.0.1", addr.getAddress().getHostAddress());
+    assertEquals(12345, addr.getPort());
+
+    // ----------------------------------------------------
```

Comment:
nit: delete no meaningless annotation?

Reply:
@Hexiaoqiao , thanks for your review.
To test the Cache, the same case is executed twice. There are a total of three cases. In order to divide the three cases, these comments are added.